packages/lesswrong/lib/collections/posts/formGroups.ts

```diff
@@ -85,7 +85,7 @@ export const formGroups: Partial<Record<string,FormGroup>> = {
   tags: {
     order: isEAForum ? 22 : 60,
     name: "tags",
-    label: `Post ${taggingNamePluralCapitalSetting.get()}`,
+    label: `Set ${taggingNamePluralSetting.get()}`,
```

Comment:
```suggestion
    label: `Apply ${taggingNamePluralSetting.get()}`,
```

Agree with @darkruby501 on this one! Let's do `Apply` instead of` Set`